pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -339,60 +336,23 @@ public void testStringLiteral() {
     Assert.assertEquals(groupbyList.get(1).getFunctionCall().getOperands().get(1).getIdentifier().getName(), "FOO");
   }
 
-  @Test
-  public void testConverter()
-      throws IOException {
-    CalciteSqlParser.compileToPinotQuery("SELECT MIN(div(DaysSinceEpoch,2)) FROM mytable");
```

Comment:
should we add these queries to the sql_queries.list? Its possible that these were added to test edge cases.

Reply:
As per the discussion here #4387, this was added as a sample for dev's to be able to debug. I had made the same comment to @Jackie-Jiang above.